data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -124,16 +124,16 @@ railway-station-subway-beijing                      # icon z12- (also has captio
 railway-station-subway-berlin                       # icon z12- (also has caption(optional) z14-)
 railway-station-subway-buenos_aires                 # icon z12- (also has caption(optional) z14-)
 railway-station-subway-istanbul                     # icon z12- (also has caption(optional) z14-)
-railway-station-subway-kiev                         # icon z12- (also has caption(optional) z12-)
+railway-station-subway-kiev                         # icon z12-16 and caption z17- (also has caption(optional) z12-16)
 railway-station-subway-la                           # icon z12- (also has caption(optional) z14-)
 railway-station-subway-lisboa                       # icon z12- (also has caption(optional) z14-)
 railway-station-subway-london                       # icon z12- (also has caption(optional) z14-)
 railway-station-subway-madrid                       # icon z12- (also has caption(optional) z14-)
 railway-station-subway-mexico                       # icon z12- (also has caption(optional) z14-)
-railway-station-subway-minsk                        # icon z12- (also has caption(optional) z12-)
+railway-station-subway-minsk                        # icon z12-16 and caption z17- (also has caption(optional) z12-16)
 railway-station-subway-newyork                      # icon z12- (also has caption(optional) z14-)
 railway-station-subway-paris                        # icon z12- (also has caption(optional) z14-)
-railway-station-subway-roma                         # icon z12- (also has caption(optional) z14-)
+railway-station-subway-roma                         # icon z12-16 and caption z17- (also has caption(optional) z14-16)
```

Comment:
these are all due to fancy styles for roma, moscow & kiev, these styles are actually what is used in the clear style now. i think the idea is that only the station entrances are shown on high zooms.

Reply:
ah ok, makes sense!
it's showing as `z12-` on master because it uses a zero-icon from z17-

still, there is a bug - on z17 there is no icon but there is a caption

Reply:
fixed